server/common/src/main/java/org/eclipse/lsp/cobol/common/symbols/SymbolTable.java

```diff
@@ -92,4 +84,24 @@ private static boolean isGlobal(VariableNode node) {
     }
     return false;
   }
+
+  public Map<String, CodeBlockReference> getParagraphMap() {
```

Comment:
I wonder if these should be call `get...MapCopy` to indicate both the perf implications and the fact that any modifications do not impact the original source (unlike `getProcedures`).